src/elements/content-uploader/OverallUploadsProgressBar.js

```diff
@@ -94,6 +94,7 @@ const OverallUploadsProgressBar = ({
             onKeyDown={onKeyDown}
             role="button"
             tabIndex={isVisible ? '0' : '-1'}
+            aria-hidden={!isVisible}
```

Comment:
nit: props should be sorted alphabetically

Reply:
I'm wondering if we should enable `react/jsx-sort-props` rule in this case? This is easy to miss if checked manually

Reply:
I like the idea but I don't what effect it's going to have on BUIE users. I think it's a topic for another issue/PR. 😄